examples/js/build.gradle.kts

```diff
@@ -0,0 +1,24 @@
+plugins {
+    id("org.jetbrains.kotlin.js") version "1.4.0"
+    id("org.jetbrains.kotlin.plugin.serialization") version "1.4.0"
+    id("com.kotlin.aws.js.js-plugin") version "0.0.1" apply true
```

Comment:
`com.kotlin.aws.js.js-plugin` strange name, probably should be `com.kotlin.aws.js.plugin`